CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# biobox 0.4.0
+
+## BREAKING CHANGES
+
+* `fq_subsample`: This component has been removed in favour of `fq/fq_subsample` (PR #182).
```

Comment:
I guess this is just a move right? Could you make that more clear (that the two are identical)?